Libraries/LibWeb/Crypto/SubtleCrypto.cpp

```diff
@@ -1212,7 +1212,7 @@ SupportedAlgorithmsMap& supported_algorithms_internal()
     return s_supported_algorithms;
 }
 
-// https://w3c.github.io/webcrypto/#algorithm-normalization-internalS
+// https://w3c.github.io/webcrypto/#algorithm-normalization-internal
```

Comment:
You've got a new typo in the commit message though: "algorith"

But I don't really understand what the commit message has to do with this change?